.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  'extends': 'airbnb',
+  'rules': {
+    'comma-dangle': ['error', 'never'],
+    'import/no-unresolved': 'off',
+    'import/no-extraneous-dependencies': 'off',
+    'import/extensions': 'off',
+    'no-console': 'off',
+    semi: ['error', 'never'],
```

Comment:
@justin808 I personally have no problem with just using airbnb standard, it's just that a couple of folks here pointed out to not have semi-colons and commas due to aesthetics, so it's been removed for now. We can always add it later.  

Sure, I see your point for `no-console`. I will fix it 👍